server/tasks/process_file.py

```diff
@@ -117,9 +117,16 @@ def process_file_streaming(self, agent: Agent, parent_job_id: str):
     agent.environment.kafka_input_topic = get_input_topic(parent_job_id)
     agent.environment.kafka_output_topic = get_output_topic(parent_job_id)
 
+    # import time
+    # time.sleep(5)
+
     # Run the agent
     asyncio.run(agent.arun())
 
+    # manually cleanup these topics
+    delete_topic(agent.environment.kafka_input_topic)
+    delete_topic(agent.environment.kafka_output_topic)
```

Comment:
Should we do this in the parent task instead of this input task?

Reply:
I want to keep it as close to where they are used as possible, we already need to get the topic name here so no sense doing it in 2 places. It could also be different for different subtasks called by the same parent task in the future (for ex, when we switch to a reusable topic pool it'll be "unlocking" the topic instead of deleting it)

Reply:
I think there is a risk here - input job is done, we delete the topic, but output job is still trying to run/pull things from output topic queue, so I would not delete it here

Reply:
oh you're right actually, I'll change it

Reply:
Not changed yet right?

Reply:
done